WcaOnRails/env_vars.rb

```diff
@@ -0,0 +1,61 @@
+# frozen_string_literal: true
+
+EnvVars = Env::Vars.new(
+  env: {
```

Comment:
I'd rather provide an `env.sample` file with these keys populated in a `KEY=value123` fashion that every individual developer has to add to their repo during bootstrap.

1. Create an `env.sample` file instead of hard-coding the values here
2. Figure out how the actual non-sample file has to be called (my guess would be `development.env`). Consult the `dotenv` gem to find out.
3. Figure out in conjunction with (2) _where_ that file has to be placed so that `dotenv` sees it.
4. Update `.gitignore` to ignore the (hypothetical) `development.env`
5. Update the README to make people aware that they will need to execute something like `cp env.sample development.env` when first setting up the repo
6. Ask questions if this list is too confusing

Reply:
A global +1 to what Gregor said, some more details:
  - for 1 and 2, I would directly create a `.env.development` file containing good defaults for devs and acting as a "sample" configuration file (and a similar file for tests if needed)! There are a few possible ways to override this locally (see [here](https://github.com/bkeepers/dotenv#what-other-env-files-can-i-use)), so people not happy with the default variables would be able to override them without issues (and I would likely include something in the readme to point people to dotenv's documentation for local overrides).
  - 3: it's in `WcaOnRails`, I'm pretty sure we have a `.env.production` file there on prod :)
  - 4: the `.gitignore` actually already ignores all `.env*` files! However I would definitely consider updating it to reflect what is recommended [here](https://github.com/bkeepers/dotenv#what-other-env-files-can-i-use). But still git-ignoring the production one because in our setup it contains sensitive data.

Reply:
Almost got dotenv working with env_vars following the instruction of you guys 😁. Now I need to name them properly according to dotenv's README, as mentioned by Philippe.